presto-main/src/test/java/com/facebook/presto/operator/repartition/BenchmarkPartitionedOutputOperator.java

```diff
@@ -274,7 +277,10 @@ private PartitionedOutputBuffer createPartitionedOutputBuffer()
 
         private OptimizedPartitionedOutputOperator createOptimizedPartitionedOutputOperator()
         {
-            PartitionFunction partitionFunction = new LocalPartitionGenerator(new PrecomputedHashGenerator(0), PARTITION_COUNT);
+            PartitionFunction partitionFunction = new BucketPartitionFunction(
```

Comment:
So this 35% improvements is end-to-end partitioned output operator performance improvement?

Reply:
Yes it's the addPage() performance

Reply:
Yes it's the end-end addPage() performance gain.